src/crypto/tests/TestPersistentStorageOpKeyStore.cpp

```diff
@@ -130,7 +129,17 @@ class TestOperationalKeystore final : public Crypto::OperationalKeystore
     FabricIndex mUsedFabricIndex = 0;
 };
 
-void TestBasicLifeCycle(nlTestSuite * inSuite, void * inContext)
+struct TestPersistentStorageOpKeyStore : public ::testing::Test
+{
+    void SetUp() override
```

Comment:
I think that it might be better to initialize memory per entire testsuite, not per test. Also, use `ASSERT_*` in SetUp* functions.

Reply:
Fixed